### ISSUE TITLE
InstagramMedia get comments count returns zero.

### ISSUE THREAD

**Issue by vigneshuvi** [shyambhat/InstagramKit#255]
Hi

You wrote the following code to get comments count and it's comment. Most of the time kData is not available and we can get only comments count. But 

```
NSDictionary *commentsDictionary = info[kComments];
        self.mComments = [[NSMutableArray alloc] init];
        if ([commentsDictionary isKindOfClass:[NSDictionary class]]) {
           **// KData is not available.**
            for (NSDictionary *commentInfo in commentsDictionary[kData]) { 
                InstagramComment *comment = [[InstagramComment alloc] initWithInfo:commentInfo];
                [self.mComments addObject:comment];
            }
            **// kCount is available.**
            NSNumber *commentsCount = commentsDictionary[kCount];
            self.commentsCount = commentsCount.integerValue;
        }

```

---

In the following getter method, you used mComments array count for getting the media's comments count. Instead of that, Can you please use "self.commentsCount" variable which will give us the perfect result..

```
- (NSInteger)commentCount
{
    return [self.mComments count]; 
}
```

https://github.com/shyambhat/InstagramKit/edit/master/InstagramKit/Models/InstagramMedia.m

**Comment by shyambhat** [comment]
This was a grave mistake - a leftover from previous edits. I spotted and fixed this just minutes ago on the dev branch. I'll include it in a release soon. Thanks for pointing out!

**Comment by vigneshuvi** [comment]
@shyambhat  Have you pushed your fix to master branch? Why you reopened ticket again? Any issues?

**Comment by shyambhat** [comment]
It's still on `dev`. Leaving it open until we release a new version, for anyone else who may spot the same issue.

**Comment by ryonandesukedo** [comment]
@shyambhat Hi, I had same problem. Could you marge the commit of dev branch to master?

**Comment by sidan5** [comment]
@shyambhat When this issue is going to be closed?

**Comment by shyambhat** [comment]
Included in v4.1.
